src/components/App/App.js

```diff
@@ -21,51 +18,43 @@ class App extends Component {
         email: '',
         purpose: ''
       },
-      isLoggedIn: true,
+      isLoggedIn: false,
     }
   }
+  
+  setUserInfo = (user) => {
+    this.setState({
+      userInfo: user,
+      isLoggedIn: true
+    })
+  }
 
-  componentDidMount() {
+  fetchAreas = async () => {
```

Comment:
IF we have time, let's see if we can move this into another file so we can do mock functions in the tests! I think that's more of a back-burner issue for right now though after we write out the bulk of our tests.